openpype/hosts/nuke/hooks/pre_nukeassist_setup.py

```diff
@@ -0,0 +1,10 @@
+from openpype.lib import PreLaunchHook
+
+class PrelaunchNukeAssistHook(PreLaunchHook):
```

Comment:
```suggestion


class PrelaunchNukeAssistHook(PreLaunchHook):
```

Reply:
what is the difference? I am perhaps blind, but it seems to be the same.